packages/api/src/modules/providers/subventia/validators/subventia.validator.test.ts

```diff
@@ -0,0 +1,196 @@
+import SubventiaValidator from "./subventia.validator";
+
+const PARSEDDATAROW = {
```

Comment:
```suggestion
const PARSED_DATA_ROW = {
```
C'est un usage de casse que de séparer les mots en majuscules par des `_` pour plus de lisibilité

Reply:
modification faites